build.sbt

```diff
@@ -128,6 +128,7 @@ lazy val commonSettings = clearSourceAndResourceDirectories ++ publishSettings +
   // we always assume that Java classes are standalone and do not have any dependency
   // on Scala classes
   compileOrder := CompileOrder.JavaThenScala,
+  incOptions := incOptions.value.withNameHashing(false).withAntStyle(true), // until we have class-based name hashing
```

Comment:
Agreed, I should've specified this commit was not meant for the PR, but I need it locally now that I no longer have `ant` on my machine :)